test/utils.js

```diff
@@ -1,3 +1,12 @@
+import serverNextAnimationFrame from './server/nextAnimationFrame';
+
 export function compactHtml(htmlStr) {
   return htmlStr.trim().replace(/>\s+</gm, `><`);
 }
+
+export function nextAnimationFrame() {
+  if (typeof requestAnimationFrame !== `undefined`) {
```

Comment:
Have you verified that you need this? `./server/nextAnimationFrame` already uses https://www.npmjs.com/package/raf which should already be a "polyfill for node and the browser"